lib/lotus/commands/new/app.rb

```diff
@@ -104,7 +104,7 @@ def upcase_app_name
         end
 
         def classified_app_name
-          Utils::String.new(app_name).classify
+          Utils::String.new(app_name).tr('-', '_').classify
```

Comment:
this should be fixed in lotus/utils repo

https://github.com/lotus/utils/blob/master/lib/lotus/utils/string.rb#L73
I think we have to add https://github.com/lotus/utils/blob/master/lib/lotus/utils/string.rb#L67